src/main/java/com/atlassian/jira/cloud/jenkins/listeners/AutoBuildsListener.java

```diff
@@ -116,32 +114,54 @@ private void maybeSendDataToJira(final boolean isOnCompleted) {
         if (finalResultSent) {
             return;
         }
-        if (issueKeyExtractor.extractIssueKeys(this.build).isEmpty()) {
+        pipelineLogger.debug("Checking for issue keys for this build ... ");
+        if (issueKeyExtractor.extractIssueKeys(this.build, pipelineLogger).isEmpty()) {
```

Comment:
Added a log event to every possible case, so we can ask a customer to activate debug mode and share the logs with us and we'll know which code was executed.